common/unsafe/src/main/java/org/apache/spark/sql/catalyst/util/DateTimeConstants.java

```diff
@@ -21,6 +21,8 @@
 
   public static final int MONTHS_PER_YEAR = 12;
 
+  public static final byte DAYS_PER_MONTH = 31;
```

Comment:
Let's **not** add this here. 31 days per month is streaming specific constant. And it is not applicable to other classes in  Spark SQL.